src/contactInfo/domain/useCases/SubmitContactInfo.ts

```diff
@@ -0,0 +1,27 @@
+import { UseCase } from '../../../core/domain/useCases/UseCase'
+import { ContactDTO } from '../dtos/ContactDTO'
+import { Contact } from '../models/Contact'
+import { IContactRepository } from '../repositories/IContactRepository'
+
+export class SubmitContactInfo implements UseCase<Contact[]> {
+  private contactRepository: IContactRepository
+
+  constructor(contactRepository: IContactRepository) {
+    this.contactRepository = contactRepository
+  }
+
+  /**
+   * Submits contact information and returns a DTO containing the submitted data.
```

Comment:
It doesn't actually return a DTO. The DTO is only used to send the necessary data to the API to create the contact. It returns a Contact model.